src/zino/snmp.py

```diff
@@ -188,6 +188,8 @@ def _object_type_to_mib_object(cls, object_type: ObjectType) -> MibObject:
             value = int(value)
         elif isinstance(value, univ.OctetString):
             value = str(value)
+        elif isinstance(value, ObjectIdentity):
+            value = value.getOid().asTuple()
```

Comment:
typing for `value` in `MibObject` should be updated to reflect this

Reply:
Done!